internal/npm_install/pre_process_package_json.js

```diff
@@ -53,9 +53,13 @@ function main() {
     // Note: there is no equivalent npm functionality to clean out individual packages
     // from the npm cache.
     clearYarnFilePathCaches(pkg);
+    console.log(isPnp(pkg));
```

Comment:
```suggestion
```

leftover?

Reply:
This script is called by bazel as a way to tell the rule that the workspace is a pnp workspace, bazel can only take from stdout, so we need to log this to the console